src/less/components/breadcrumb/index.less

```diff
@@ -1,17 +1,22 @@
 /* ========== hxBreadcrumbs ========== */
 .hxBreadcrumb {
+  display: flex;
+  margin-top: 0.5rem;
+  margin-bottom: 1rem;
   align-content: center;
   align-items: center;
-  display: flex;
+  line-height: 1rem;
   font-size: 0.75rem;
   font-weight: 500;
-  height: 1.75rem;
-  margin-bottom: 1rem;
-  text-transform: uppercase;
+  text-transform: lowercase;
```

Comment:
This property is unnecessary. We can apply `text-transform: capitalize;` to the links instead. (see below).

Reply:
Done as per your comment.

My original thought was to apply lowercase to parent element and capitalize to the links. So that users are not free to enter typos like "Child ELement" etc. Somehow, that slipped from my mind while testing different text strings in breadcrumb and did a wrong fix yesterday.

Now, I just simply applied text-transform: capitalize for links and removed text-transform for parent and everywhere as you suggested.